README.rst

```diff
@@ -0,0 +1,115 @@
+Introduction
+============
+
+``rq-dashboard`` is a general purpose, lightweight, `Flask`_-based web
+front-end to monitor your `RQ`_ queues, jobs, and workers in realtime.
+
+|Can I Use Python 3?|
+
+
+It looks like this
+------------------
+
+|image1| |image2|
+
+
+Installing
+----------
+
+.. code:: console
+
+    $ pip install rq-dashboard
+
+
+Running the dashboard
+---------------------
+
+You can either run the dashboard standalone, like this:
+
+.. code:: console
+
+    $ rq-dashboard
+    * Running on http://127.0.0.1:9181/
+    * Restarting with reloader
+    ...
+
+
+Integrating the dashboard in your Flask app
+-------------------------------------------
+
+Or you can integrate the dashboard in your own `Flask`_ app, like this:
+
+.. code:: python
+
+    from flask import Flask
+    from rq_dashboard import RQDashboard
+
+
+    app = Flask(__name__)
+    RQDashboard(app)
+
+    @app.route("/")
+    def hello():
+        return "Hello World!"
+
+    if __name__ == "__main__":
+        app.run()
+
+This will register the dashboard on the ``/rq`` URL root in your Flask
+app. To use a different URL root, use the following:
+
+.. code:: python
+
+    RQDashboard(app, url_prefix='/some/other/url')
+
+
+Making a release
+----------------
+
+The development environment assumes you are in a virtualenv and have pulled in
+the necessary build time (and run time) dependencies with
+
+::
+
+    $ pip install -r requirements.txt
+
+
+See the pip_docs_on_requirements_ and in particular the setup_vs_requirements_
```

Comment:
Ahhh, now I see. You're using requirements.txt for _development_ tools here. I thought you wanted to use it to install rq-dashboard itself. This all makes perfect sense to me :)